2.1/runtime/stretch/amd64/Dockerfile

```diff
@@ -7,11 +7,10 @@ RUN apt-get update \
 
 # Install .NET Core
 ENV DOTNET_VERSION 2.1.0-preview1-25919-02
-ENV DOTNET_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-$DOTNET_VERSION-linux-x64.tar.gz
-ENV DOTNET_DOWNLOAD_SHA E215DFD387C569707805BB693B7B006B124A2A13AA6F2C2CD9DCC8C1B34ED221A09302BC6D337276BB6CD6D13CC7A2675DE9B7BCF956D69C373D9B177446E14B
 
-RUN curl -SL $DOTNET_DOWNLOAD_URL --output dotnet.tar.gz \
-    && echo "$DOTNET_DOWNLOAD_SHA dotnet.tar.gz" | sha512sum -c - \
+RUN curl -SL --output dotnet.tar.gz https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-$DOTNET_VERSION-linux-x64.tar.gz \
```

Comment:
Not related to your change. But a clarifying question -  How do you determine whether to use `curl` or `wget`?

Reply:
`curl` has been the standard used in the .NET Dockerfiles however it is not available in the Alpine base image therefore `wget` is used for Alpine